src/app/bindings.ts

```diff
@@ -0,0 +1,36 @@
+import {AppBinding, AppState, AppsState} from 'mattermost-redux/types/apps';
+
+class Bindings {
```

Comment:
This class stores no state in `this`, so we can get rid of the class and use flat functions, unless we expect to provide values in a `constructor`

Reply:
Agree here too.  This file should be very simple. It only returns an array of bindings.  Simply bind a location to a call.

Reply:
We may have different bindings based on if the user is connected to Zendesk for instance. Maybe we do want it in a class if it needs to have user-related information at runtime. Alternatively, it can accept that information as function arguments.